src/bundle/DependencyInjection/Compiler/AuthenticationProviderDecoratorCompilerPass.php

```diff
@@ -15,6 +24,9 @@
  */
 class AuthenticationProviderDecoratorCompilerPass implements CompilerPassInterface
 {
+    /**
```

Comment:
That's correct, but technically it would be a BC break here because the class is not final, so someone might extend it (even if it's unlikely in this case).

Reply:
I had some trouble extending classes or implementing interfaces when the method signature is not identical the the parent one. TBD